### DISCUSSION TITLE
How do you actually make the .exe?

### DISCUSSION THREAD

**Discussion by RossPfeiffer** [atom/atom#22234] [Q&A]
I've got the electron-quick-start example, but how do you turn that into an .exe?
People that want to use my softare don't want to have to open the command line each time and run this like a hacker.
What's it called to push this thing into production?

nvm. googled
sometimes it helps just to ask so i can hear how obvious the answer would be

**Comment by sadick254** [comment]
A good place to start would be https://www.electronjs.org/docs/tutorial/application-distribution
